### PR TITLE
change apiRepr() to serialize(), mocha version bump

### PR DESCRIPTION
**Pull request by tsongas**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node server.js",
-    "test": "mocha"
+    "test": "mocha --exit"
```
Comment (benjaminEwhite) [inline package.json:8]: Not sure why this is showing up in the diff -- master already has the `--exit` flag.

File: package.json [hunk 2]
```diff
@@ -26,6 +26,6 @@
     "chai": "^4.0.1",
     "chai-http": "^3.0.0",
     "faker": "^3.1.0",
-    "mocha": "^3.4.2"
+    "mocha": "^4.0.1"
```
Comment (benjaminEwhite) [inline package.json:29]: ditto (also NBD, i just don't understand why it's showing up).

**Comment by benjaminEwhite** [comment]
This is ready to ship, but should go out immediately after[ Node curric changes](https://github.com/Thinkful-Ed/curric-node-001-v5/pull/455) go out

File: package.json (hunk 1 above)
Comment (tsongas) [reply package.json:8]: It must be showing a diff of what changed before vs after within my branch, not my branch vs master. Does that make sense? My best guess anyway.

Comment (tsongas) [reply package.json:8]: I forked master before you added the exit flag and bumped the mocha version, then I made those same changes in my branch.

**Pull request merged**